sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTest.java

```diff
@@ -167,17 +198,21 @@ void recognizeLayoutSourceUrl() {
     /**
      * Verifies that an exception is thrown for invalid source url for recognizing content information.
      */
-    @Test
-    void recognizeLayoutInvalidSourceUrl() {
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.formrecognizer.TestUtils#getTestParameters")
+     public void recognizeLayoutInvalidSourceUrl(HttpClient httpClient, FormRecognizerServiceVersion serviceVersion) {
+        client = getFormRecognizerClient(httpClient, serviceVersion);
```

Comment:
does this mean we will create a new client every time before a test?

Reply:
Yes, it is a cheap operation and parameterized tests can only set on method level instead of @beforeAll and @before